webapp/src/Command/ImportEventFeedCommand.php

```diff
@@ -1161,7 +1086,7 @@ protected function importOrganization(array $event)
      * @param array $event
```

Comment:
I think you can remove these for this file quite often.

Reply:
Didn’t I push this change already?

Reply:
The file is not outdated yet (in my view) so you might have it locally.

Reply:
its now outdated.